.golangci.yml

```diff
@@ -11,6 +11,7 @@ linters:
   enable:
     - depguard
     - errcheck
+    - exportloopref
```

Comment:
```suggestion
```

The problem that `exportloopref` found seems fixed after Go 1.22. Do you still need this linter? https://go.dev/blog/loopvar-preview